extras/rya.streams/api/src/main/java/org/apache/rya/streams/api/entity/StreamsQuery.java

```diff
@@ -60,17 +63,25 @@ public String getSparql() {
         return sparql;
     }
 
+    /**
+     * @return {@code true} if Rya Streams should process this query; otherwise {@code false}.
+     */
+    public boolean isActive() {
+        return isActive;
+    }
+
     @Override
     public int hashCode() {
-        return Objects.hash(queryId, sparql);
+        return Objects.hash(queryId, sparql, isActive);
     }
 
     @Override
     public boolean equals(final Object o) {
         if(o instanceof StreamsQuery) {
             final StreamsQuery other = (StreamsQuery) o;
             return Objects.equals(queryId, other.queryId) &&
```

Comment:
Use EqualsBuilder

Reply:
Why? This requires less object construction, so I'm not going to do that.